src/braket/circuits/gate.py

```diff
@@ -46,8 +46,13 @@ def __init__(self, qubit_count: Optional[int], ascii_symbols: Sequence[str]):
             ValueError: `qubit_count` is less than 1, `ascii_symbols` are `None`, or
                 `ascii_symbols` length != `qubit_count`
         """
+        # todo: implement ascii symbols for control modifier
         super().__init__(qubit_count=qubit_count, ascii_symbols=ascii_symbols)
 
+    @property
+    def _qasm_name(self):
```

Comment:
Might actually be worth making this a public property, so users can see _what_ a gate gets serialized as; it's also easier to miss and get confused over when adding new gates.

Ideally, this would actually _be_ the `name` property.

Reply:
Makes sense to me, I'll go ahead and do that in a follow-up pr